src/pytest_dbfixtures/factories/postgresql.py

```diff
@@ -233,7 +233,10 @@ def postgresql_factory(request):
         """
         proc_fixture = request.getfuncargvalue(process_fixture_name)
 
-        psycopg2, config = try_import('psycopg2', request)
+        if platform.python_implementation() == 'PyPy':
```

Comment:
I don't like this, while I'm not sure if there is a more typical way to run an available package among several supported ones.